bin/templates/cordova/lib/build.js

```diff
@@ -422,11 +500,18 @@ function parseOpts(options, resolvedTarget) {
 module.exports.runClean = function(options) {
     var opts = parseOpts(options);
     var builder = builders[opts.buildMethod];
-    return builder.prepEnv()
+    return builder.prepEnv(opts)
     .then(function() {
-        return builder.clean(opts.extraArgs);
+        return builder.clean(opts);
     }).then(function() {
         shell.rm('-rf', path.join(ROOT, 'out'));
+
+        ['debug', 'release'].forEach(function(config) {
+            var propertiesFilePath = path.join(ROOT, config + SIGNING_PROPERTIES);
+            if(isAutoGenerated(propertiesFilePath)){
+                shell.rm('-f', propertiesFilePath);
```

Comment:
My main rationale was that these .properties file have sensitive information including passwords and I wanted developers to have a way to clean this if needed. Invoking the 'clean' script will run this and delete these files.